grub-core/kern/tpm.c

```diff
@@ -0,0 +1,13 @@
+#include <grub/err.h>
+#include <grub/i18n.h>
+#include <grub/misc.h>
+#include <grub/mm.h>
+#include <grub/tpm.h>
+#include <grub/term.h>
+
+grub_err_t
+grub_tpm_measure (unsigned char *buf, grub_size_t size, grub_uint8_t pcr,
+		  const char *description __attribute__ ((unused)))
```

Comment:
The unused attribute is left over from me not having implemented the logging support yet. Oops. The descriptions don't factor into the hash, they're intended to be parsed by the attestation layer so the hashes can be replayed and verified against policy.